cmake.md

```diff
@@ -0,0 +1,30 @@
+## CMake build system
```

Comment:
I would probably  wrap the all caps names in backticks. This prevents any weirdness around underscores getting parsed as underline and makes it monospace / stand out as something special.

Reply:
Done